web/src/views/repo/RepoWrapper.vue

```diff
@@ -38,7 +38,7 @@
     <Tab id="activity" :title="$t('repo.activity')" />
     <Tab id="branches" :title="$t('repo.branches')" />
     <Tab
-      v-if="config.forge === 'gitea' || config.forge === 'github' || config.forge === 'gitlab'"
+      v-if="config.forge === 'gitea' || config.forge === 'github' || config.forge === 'gitlab'  || config.forge == 'stash'"
```

Comment:
Wouldn't this should also show pull-requests for the Bitbucket cloud forge? Removing the condition? But I noticed I did `==` instead of `===` must be getting rusty in JS/TS :)